modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java

```diff
@@ -42,12 +42,10 @@ public PostgreSQLContainer(final String dockerImageName) {
 
     @Override
     protected void configure() {
-
         addExposedPort(POSTGRESQL_PORT);
         addEnv("POSTGRES_DB", databaseName);
         addEnv("POSTGRES_USER", username);
         addEnv("POSTGRES_PASSWORD", password);
-        setCommand("postgres");
```

Comment:
I just checked the official Dockerfile and the default CMD is already `postgres`, so I'm wondering why we have this here? Do you have some background @rnorth?

Reply:
No, to be honest I think this was just present by mistake. As long as our tests continue to pass with it removed, I'd be happy for it to go.